Presentation.Web/app/components/org/structure/org-unit-migration-table.view.html

```diff
@@ -0,0 +1,37 @@
+﻿<div class="panel panel-default">
+    <div class="panel-heading">
+        <a id="expand_collapse_registrations" class="task-fold control-label" data-ng-click="isHidden = !isHidden">
+            <span>{{ctrl.title}}</span>
+            <i class="faded-gray fa pull-right" data-ng-class="{'fa-minus-square-o': !isHidden, 'fa-plus-square-o': isHidden}"></i>
+        </a>
+    </div>
+    <div ng-if="!isHidden" class="panel-body">
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th class="col-xs-1">
+                    <input type="checkbox" ng-model="ctrl.root.selected" ng-change="ctrl.registrationGroupSelected(ctrl.root)" />
+                </th>
+                <th ng-repeat="header in ctrl.configuration">{{header.title}}</th>
+                <th class="col-xs-1">Slet</th>
+            </tr>
+            </thead>
+            <tbody ng-repeat="registration in ctrl.root.children">
+            <tr>
+                <td><input data-disabled-on-busy type="checkbox" ng-model="registration.selected" ng-change="ctrl.registrationSelected()" /></td>
+                <td ng-repeat="header in ctrl.configuration">
+                    <div class="{{header.cssClass}}">
+                        <a ng-if="header.isLink" href="" ng-click="ctrl.navigateTo(registration.targetPageObjectId)">
```

Comment:
When I tested this I think it would work better if we just render an sref and then with target "_blank".. Then the user can actually jump to the registration and work on it without being unable to jump back to the edit dialog.